provider/resource_playbook.go

```diff
@@ -470,23 +470,14 @@ func resourcePlaybookUpdate(data *schema.ResourceData, meta interface{}) error {
 
 	log.Printf("Temp Inventory File: %s", tempInventoryFile)
 
-	// Get all available temp inventories and pass them as args
-	inventories := providerutils.GetAllInventories(inventoryFileNamePrefix)
-
 	log.Print("[INVENTORIES]:")
 	log.Print(inventories)
```

Comment:
Hi, thank you for contributing :)
I will approve and merge this PR once all the tests pass. I approved workflow (tests) run for this PR and both of the tests failed. I think removing the lines I've marked should make the tests pass.

Reply:
Done :)